source/_data/plugins.yml

```
@@ -2698,3 +2698,12 @@
     - html-minifier
     - uglify
     - clean-css
+- name: hexo-douban-card
+  description: generate a douban book(movie/music) card in hexo post 
+  link: https://github.com/TankNee/hexo-douban-card
+  tags:
+    - douban
+    - post
+    - card
+    - crawl
+    - spider
```

```suggestion
    - spider

```

Would you mind adding an extra blank line at the EOF?

---

ok,done